server/app/api/v1/charity/controller.js

```diff
@@ -0,0 +1,108 @@
+import {
+  NotFoundError, AlreadyInUseError, ValidationError
+} from 'common-errors';
+import merge from 'lodash/merge';
+import Charity from './model';
+
+import { VALID_OBJECT_ID } from '../../../util/validations';
```

Comment:
I had intended to write some validation functions in that file at some point which is why I named it that. Didn't get around to it on this PR. Good call though!